src/schema.js

```diff
@@ -89,12 +89,26 @@ const postSchema = Joi.object({
 		.required(),
 	image: Joi.string()
 		.alphanum(),
-	content: Joi.string(),
+	content: Joi.string()
+		.required(),
+	published: Joi.boolean()
+})
+
+const updatePostSchema = Joi.object({
+	title: Joi.string()
+		.min(3)
+		.max(30)
+		.required(),
+	image: Joi.string()
+		.uri(),
+	article: Joi.string()
+		.required(),
 	published: Joi.boolean()
```

Comment:
for context, like you might have seen, a `req` object has `body`, `params` and so on so now since you we're validating the req as a whole you have to specify the part of the req you're validating like I've done below

```suggestion
const updatePostSchema = Joi.object({
       body: {
          title: Joi.string()
		.min(3)
		.max(30)
		.required(),
	image: Joi.string()
		.uri(),
	article: Joi.string()
		.required(),
	published: Joi.boolean()
       }
       })
```